Library/Homebrew/attestation.rb

```diff
@@ -246,6 +248,15 @@ def self.check_core_attestation(bottle)
       end
 
       backfill_attestation
+    rescue InvalidAttestationError
+      @attestation_retry_count ||= T.let(Hash.new(0), T.nilable(T::Hash[Bottle, Integer]))
+      raise if @attestation_retry_count[bottle] >= ATTESTATION_MAX_RETRIES
+
+      sleep_time = 3 ** @attestation_retry_count[bottle]
```

Comment:
```suggestion
      sleep_time = 2 ** @attestation_retry_count[bottle]
```
would be fine with me if you want this to take smaller jumps/be quicker to fail